board-viewer.html

```diff
@@ -1653,14 +1652,16 @@
       var filters = [];
       for (var i = 0; i < this._filters.length; i++) {
         if (this._filters[i].name.indexOf("connected-") == 0 ||
-            this._filters[i].name.indexOf("highlit-") == 0 ||
+            this._filters[i].name.indexOf("highlight-") == 0 ||
             this._filters[i].name.indexOf("related-") == 0) {
```

Comment:
@juliocri Can you put this "highlit" fix into a separate PR so it doesn't get lost? I was going to close this PR when I saw this fix was part of it.

Thanks,
James